omplConfig.cmake.in

```diff
@@ -1,63 +1,40 @@
-# CMake OMPL module
+# CMake OMPL config module
 #
-# It defines the following variables:
-# OMPL_FOUND         - TRUE
-# OMPL_INCLUDE_DIRS  - The OMPL include directories
-# OMPL_LIBRARY_DIRS  - The OMPL library directories
-# OMPL_LIBRARIES     - The OMPL library
-# OMPLAPP_LIBRARIES  - The OMPL.app libraries (if installed)
-# OMPL_VERSION       - The OMPL version in the form <major>.<minor>.<patchlevel>
-# OMPL_MAJOR_VERSION - Major version
-# OMPL_MINOR_VERSION - Minor version
-# OMPL_PATCH_VERSION - Patch version
+# Usage:
+# find_package(ompl @OMPL_VERSION_MAJOR@ REQUIRED)
+# add_library(your_app main.cpp)
+# target_link_libraries(your_app PRIVATE ompl::ompl)
 
 @PACKAGE_INIT@
 
-set(OMPL_VERSION @PROJECT_VERSION@)
-set(OMPL_MAJOR_VERSION @PROJECT_VERSION_MAJOR@)
-set(OMPL_MINOR_VERSION @PROJECT_VERSION_MINOR@)
-set(OMPL_PATCH_VERSION @PROJECT_VERSION_PATCH@)
+include ("${CMAKE_CURRENT_LIST_DIR}/omplExport.cmake" )
+include(CMakeFindDependencyMacro)
+find_dependency(Boost REQUIRED COMPONENTS serialization filesystem system)
+find_dependency(Eigen3 REQUIRED)
 
-set_and_check(OMPL_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
-set(OMPL_INCLUDE_DIRS "${OMPL_INCLUDE_DIR};@Boost_INCLUDE_DIR@;@EIGEN3_INCLUDE_DIR@")
```

Comment:
In trying to bring back the legacy variables, I found this would have only worked if this package was BUILT with boost in the same location as the where it was FOUND when used. This does not allow for previous OMPL to robust to a custom location for boost to show up. 

Now that we link to Boost:::<target>, those concerns go away.